### PR TITLE
Add fab command to start the python server.

### PR DESCRIPTION
**Pull request by Kapin**
Just makes it easier to run from the root folder

### CODE REVIEW

File: fabfile.py [hunk 1]
```diff
@@ -24,6 +24,11 @@ def verify():
     test()
 
 
+def runserver():
+    local('python src-backend/manage.py syncdb --noinput')
```
Comment (Ommy) [inline fabfile.py]: Is `--noinput` just not specifying a migration file?

Comment (Kapin) [inline fabfile.py]: uhhh, I think it just ensures that we don't have to type anything. I'm just following the rest of the fab commands.

Comment (Ommy) [inline fabfile.py]: Yep, cool

https://docs.djangoproject.com/en/1.8/ref/django-admin/#django-admin-option---noinput

**Comment by Ommy** [comment]
lgtm

**Pull request merged**